cvat/schema.yml

```diff
@@ -1531,6 +1531,64 @@ paths:
       responses:
         '204':
           description: The invitation has been deleted
+  /api/invitations/{key}/accept:
+    post:
+      operationId: invitations_create_accept
```

Comment:
Why is this called `invitations_create_accept`? Doesn't seem like it would create an invitation.

Reply:
Fixed